packages/core-bridge/scripts/build.js

```diff
@@ -39,28 +40,13 @@ function compile(target) {
       throw err;
     }
   }
-  const { status, error } = spawnSync(
-    'cargo-cp-artifact',
-    [
-      '--artifact',
-      'cdylib',
-      'temporal_sdk_typescript_bridge',
-      out,
-      '--',
-      'cargo',
-      'build',
-      '--message-format=json-render-diagnostics',
-      '--release',
-      ...(target ? ['--target', target] : []),
-    ],
-    { stdio: 'inherit' }
-  );
-  if (error !== undefined) {
+
+  try {
+    argv = ['--artifact', 'cdylib', 'temporal_sdk_typescript_bridge', out, '--', 'cargo', 'build', '--message-format=json-render-diagnostics', '--release', ...(target ? ['--target', target] : []), ]
+    cargo_cp_artifact(argv, process.env)
```

Comment:
After reading the code for this package: https://github.com/neon-bindings/cargo-cp-artifact/blob/main/src/index.js
I'm convinced you should await here.

Reply:
I am not a js pro, but I disagree. The exported run() is not declared async. The "bin/.." script does not use an await itself... But I could be wrong ;-)

Reply:
Right, I took another look and it seems like this tool isn't meant to be imported.
You have to run it as a child process.

```js
spawnSync(process.argv[0], [which.sync('cargo-cp-artifact'), /* rest of the args */]`)
```
https://www.npmjs.com/package/which

Reply:
used "which" and "spawnSync", removed direct call